spec/lib/tasks/workers_rake_spec.rb

```diff
@@ -39,6 +39,15 @@
     it_behaves_like 'a queued job worker', :expected_job_class
   end
 
+  describe 'workers:complete_upload:run' do
```

Comment:
This, as well as the above four describe blocks, are not needed since the `JobsRunner.workers_registry.each` loop below automatically tests all the individual worker tasks. Sorry for the confusion this legacy code has created!